apiserver/params/storage.go

```diff
@@ -33,3 +33,28 @@ type StorageShowResult struct {
 type StorageListResult struct {
 	Instances []StorageInstance
 }
+
+// PoolInstance holds data for a pool instance.
+type PoolInstance struct {
+	Name   string
+	Type   string
+	Traits map[string]interface{}
```

Comment:
M a bit allergic to config as a var name, especially since we've got more than a few of them. However, I agree that here consistency is important - renamed :D